.travis.yml

```diff
@@ -14,7 +14,8 @@ before_install:
   - ./travis-tool.sh bootstrap
 
 install:
-  - ./travis-tool.sh install_aptget r-cran-rcurl r-cran-googlevis r-cran-zoo r-cran-rcolorbrewer r-cran-ggplot2 r-cran-rvest
+  - ./travis-tool.sh install_aptget r-cran-googlevis r-cran-zoo r-cran-ggplot2 r-cran-rvest r-cran-rcpp r-cran-bh
+  - ./travis-tool.sh install_r anytime knitr rmarkdown
```

Comment:
Knitr and rmarkdown are also available as r-cran-* variants, and install faster that way.

I asked Michael to prepare r-cran-anytime.  Running the [rocker/r-apt:trusty](https://github.com/rocker-org/rocker/tree/master/r-apt/trusty) Docker container now to check...